src/components/auth/SignInForm.tsx

```diff
@@ -15,7 +15,7 @@ type SignInFormProps = {
 
 export const SignInForm = ({ onSubmit, signInError }: SignInFormProps) => (
   <Formik
-    initialValues={{ email: '', password: '' }}
+    initialValues={{ email: 'd@d.com', password: 'dddddddd' }}
```

Comment:
you probably forgot to leave this as it was before
```suggestion
    initialValues={{ email: '', password: '' }}
```